metron-analytics/metron-profiler-client/src/test/java/org/apache/metron/profiler/client/ProfileWriter.java

```diff
@@ -92,8 +92,8 @@ private void write(ProfileMeasurement m, List<Object> groups) {
     byte[] rowKey = rowKeyBuilder.rowKey(m, groups);
     ColumnList cols = columnBuilder.columns(m);
 
-    List<Mutation> mutations = hbaseClient.constructMutationReq(rowKey, cols, Durability.SKIP_WAL);
-    hbaseClient.batchMutate(mutations);
+    hbaseClient.addMutation(rowKey, cols, Durability.SKIP_WAL);
```

Comment:
I don't think you want to `SKIP_WAL`  That will result in data loss on region server crash.